mango/dashboards/file_explorer_app.py

```diff
@@ -574,7 +574,7 @@ def _open_html():
                             df,
                             use_container_width=True,
                             num_rows="dynamic",
-                            key=key_tab,
+                            key=f"{key_tab}_{uuid.uuid4()},
```

Comment:
This line solves the rendering problem but prevents the saving of files. I'm going to check this part to get everything right.